object/types.proto

```diff
@@ -160,3 +160,21 @@ message Object {
   // Payload bytes.
   bytes payload = 4 [json_name = "payload"];
 }
+
+// Meta information of split hierarchy for object assembly. With right child
+// one can traverse linked list back to left or first child to assembly
+// original object. With link object one can assembly object straight
+// away from the object parts.
+message SplitInfo {
+  // 16 byte UUID used to identify the split object hierarchy parts.
+  bytes split_id = 1;
+
+  // Identifier of most right child in split hierarchy parts. It contains
+  // split header with original object header.
+  neo.fs.v2.refs.ObjectID right_child_id = 2;
+
+  // Identifier of linking object for split hierarchy parts. It contains
+  // split header with original object header and sorted list of
+  // object parts.
+  neo.fs.v2.refs.ObjectID link_id = 3;
```

Comment:
Maybe just 'link' here? It's ob ObejctID type, mabe there is no need to add `_id` suffix?

Reply:
Fixed.